submatrix/homeserver.yaml.template

```diff
@@ -0,0 +1,37 @@
+server_name: "${MATRIX_HOSTNAME}"
+pid_file: /homeserver.pid
+listeners:
+  - port: 8008
+    tls: false
+    type: http
+    bind_addresses: ['::1', '127.0.0.1']
+    resources:
+      - names:
+        - client
+        compress: false
+  - path: /run/socks/submatrix.sock
+    type: http
+    x_forwarded: true
+    resources:
+      - names:
+        - client
+        - federation
+        compress: true
+database:
+  name: sqlite3
+  args:
+    database: /var/synapse/homeserver.db
+log_config: "/homeserver.log.config"
+max_upload_size: 100M
+media_store_path: /var/synapse/media_store
+registration_shared_secret_path: /var/synapse/registration_shared
+report_stats: true
+signing_key_path: "/etc/synapse/${MATRIX_HOSTNAME}.signing.key"
```

Comment:
this path isn't in a volume. Should it be?

Reply:
not sure but I suppose it can't hurt